src/Unic.ErrorManager.Website/sitecore modules/Web/Error Manager/404.aspx.cs

```diff
@@ -31,7 +31,10 @@ public partial class _404 : BaseError
         protected override void OnLoad(EventArgs e)
         {
             // set properties
-            base.SettingsKey = "ItemNotFoundUrl";
+            var isMedia = Context.Items.Contains("IsMedia")
+                ? Convert.ToBoolean(Context.Items["IsMedia"])
```

Comment:
There is a potential exception, if the value is not representing a boolean value.

Reply:
Yes, agreed. 
I saw your fixes